commands/audit/sca/swift/swift.go

```diff
@@ -90,42 +91,76 @@ func parseSwiftLine(line, directDependencyName, directDependencyVersion, descrip
 	return foundDependency, tempIndex, startLine, startCol
 }
 
+func updateDependency(content, name, version, fixVersion string) string {
+	urlPattern := `(?:https://|http://|sso://)?` + regexp.QuoteMeta(strings.TrimSuffix(name, ".git")) + `(?:\.git)?`
+	pattern := `\.package\(url:\s*"` + urlPattern + `",\s*(exact:\s*"(` + version + `)"|from:\s*"(` + version + `)"|"([\d\.]+)"\.\.\s*<?\s*"([\d\.]+)")\)`
+	re := regexp.MustCompile(pattern)
+	result := re.ReplaceAllStringFunc(content, func(match string) string {
+		submatches := re.FindStringSubmatch(match)
+		if len(submatches) == 0 {
+			return match
+		}
+
+		// Handle exact match
+		if len(submatches) > 1 && strings.Contains(submatches[1], "exact") {
+			log.Logger.Debug("Fixing dependency", name, "from version", submatches[2], "to", fixVersion)
+			return strings.Replace(match, submatches[2], fixVersion, 1)
+		}
+		// Handle from match
+		if len(submatches) > 1 && strings.Contains(submatches[1], "from") {
+			log.Logger.Debug("Fixing dependency", name, "from version", submatches[3], "to", fixVersion)
+			return strings.Replace(match, submatches[3], fixVersion, 1)
+		}
+
+		// Handle range case
+		if len(submatches) > 5 && submatches[4] != "" && submatches[5] != "" {
+			// submatches[4] is the start of the range
+			// submatches[5] is the end of the range
+			startVersion := submatches[4]
+			endVersion := submatches[5]
```

Comment:
```suggestion
			startRangeVersion := submatches[4]
			endRangeVersion := submatches[5]
```
no need for comment if variable name is descriptive

Reply:
ok